TagScriptEngine/block/helpers.py

```diff
@@ -100,11 +100,12 @@ def helper_split(
     args = (maxsplit,) if maxsplit is not None else ()
     if "|" in split_string:
         return SPLIT_REGEX.split(split_string, *args)
-    if easy and "~" in split_string:
-        return split_string.split("~", *args)
-    if easy and "," in split_string:
-        return split_string.split(",", *args)
-    return
+    if easy:
+        if "~" in split_string:
+            return split_string.split("~", *args)
+        if "," in split_string:
+            return split_string.split(",", *args)
+    return split_string
```

Comment:
The helper method is only supposed to return a list of strings. This change breaks the behavior of other blocks because of the new return type.

Reply:
I would modify the `set_footer `method to handle when `helper_split `returns `None`, rather than modifying the method itself.